.github/workflows/build.yaml

```diff
@@ -36,15 +46,18 @@ jobs:
         if: github.event_name == 'release'
         run: echo "RELEASE_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
 
+      - name: Get version
+        run: echo "VERSION_TAG=$(git describe --tags)" >> $GITHUB_ENV
+
       - name: Build Go binary amd64
-        run: go build -ldflags "-s -w -X main.GitCommit=$GITHUB_SHA -X main.GitRef=$GITHUB_REF -X main.Version=${RELEASE_TAG:-commit-$GITHUB_SHA}" -o bin/castai-cluster-controller-amd64 .
+        run: go build -ldflags "-s -w -X main.GitCommit=$GITHUB_SHA -X main.GitRef=$GITHUB_REF -X main.Version=${RELEASE_TAG:-$VERSION_TAG}" -o bin/castai-cluster-controller-amd64 .
```

Comment:
Why are you using tag instead of sha?  Commit SHA allows to build the image for PR

Reply:
It's output from `git describe --tags`.

When I used the image with commit sha, then it was not possible schedule actions, because the cluster-controller was marked as outdated.